lib/web/apiserver_test.go

```diff
@@ -2524,6 +2524,45 @@ func TestClusterKubesGet(t *testing.T) {
 	}, resp.Items[0])
 }
 
+func TestClusterDesktopsGet(t *testing.T) {
```

Comment:
Should this be on resources_test.go? Curious why this one seems to be placed apart from the others.